examples/util_pyinstaller/README.md

```diff
@@ -53,3 +53,20 @@ You can test it by running:
 ```shell
 .\dist\main.exe
 ```
+
+## Building the Executable (Linux/macOS)
+
+```shell
+cd examples/util_pyinstaller/
+pdm install
+./build.sh
+```
+
+Note to linux users that you may need to use `sudo`
```

Comment:
```suggestion
Note to Linux users that you may need to use `sudo`.
```

Reply:
I've just removed this line. Linux users will know what to do